code/modules/uplink/uplink_items.dm

```diff
@@ -1688,6 +1704,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 15
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/badass/normalclownclumsinessinjector //clown ops can buy this too, but it's in the pointless badassery section for them
```

Comment:
bad name and this shouldn't be in uplink_item/badass

Reply:
nice catch